Practice/kruglov/pep8task.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import os
+import hashlib
+import ast
+import argparse
+
+from time import *
+
+# Class names must begin with a capital letter
+class shuffler:
+
+    def __init__(self):
+        self.map = {}
+
+    def rename(self, dirname, output):
+        # Two extra spaces
+          mp3s = []
+        for root, directories, files in os.walk(dirname):
+            for file in files:
+                if file[-3:] == '.mp3':
+                    mp3s.append([root, file])
+        for path, mp3 in mp3s:
+            # Failed variable name "hashname"

```

Comment:
Why is it failed? What style rule does this "hashname" contradict? PyCharm may highlight probable spelling mistakes, but it's not a style issue.

Reply:
Ok i understand. Fixed that.